WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormViewModel+SaveTests.swift

```diff
@@ -68,6 +68,49 @@ final class ProductFormViewModel_SaveTests: XCTestCase {
         XCTAssertEqual(savedProduct, EditableProductModel(product: product.copy(statusKey: ProductStatus.pending.rawValue)))
     }
 
+    func test_adding_a_product_remotely_replaces_local_product_ID_in_productImagesUploader() throws {
```

Comment:
nit: how about `test_adding_a_product_remotely_fires_ replaceLocalID_in_productImagesUploader` to be more consistent with other test cases below? from the name, it might seem like the test case is asserting the product ID matches the new ID

Reply:
Done in 79a477c2fa91a5e5accc7ed2e827b6b02b38bb13